### PR TITLE
Feature/fhtml 4 chart switch button

### PR DESCRIPTION
**Pull request by Yinuo-M**
- Add the switch between chart/table button to the ChartBlock component. Add appropriate JS to toggle the text on the button and the visibility of chart/table on button click.
- CSS margin/padding fix for ChartBlock buttons to make it look good on mobile.
- Add the ChartBlock component to the simple article page.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [ ] your work is in a branch based on and synced with the develop branch.
- [ ] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [ ] your newComponent is imported in the index.js file.
- [ ] your folder hierarchy follows the proper structure.
- [ ] your code follows the coding standards and naming conventions defined.
- [ ] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [ ] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [ ] your code has been tested in all the different screensizes defined.
- [ ] you have run the `yarn lint` command and fixed any linting errors.
- [ ] you have updated the CHANGELOG.md file with your changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,10 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Upcoming release] - 1971-11-03
 
 ### Added
-- 
+- Switch between table/data view button for ChartBlock
```
Comment (c-ancia) [inline CHANGELOG.md:12]: Can you actually duplicate this empty template below it, add your changes in this new one and leave the top one empty please? it's just so we always have an example ready to be duplicated :)

File: src/components/article/ChartBlock/chartBlock.scss [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
 
 .chart-block {
+  margin-top: 3rem;
+
+  &__switch-button {
+    white-space: nowrap;
+    margin: 2rem 0 1rem;
+
+    @media screen and (min-width: $fsa-lg) {
+      margin-bottom: 2rem;
```
Comment (c-ancia) [inline src/components/article/ChartBlock/chartBlock.scss:11]: remember we use margin-top only :) can you please change?

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/components/article/ChartBlock/chartBlock.scss (hunk 2 above)
Comment (Yinuo-M) [reply src/components/article/ChartBlock/chartBlock.scss]: I forgot! Done

File: CHANGELOG.md (hunk 1 above)
Comment (Yinuo-M) [reply CHANGELOG.md:33]: Done

**Review by c-ancia — APPROVED** [review]

**Pull request merged**